server/src/server.ts

```diff
@@ -11,6 +11,16 @@ const PORT=3000
 //* this will take the plain text input and change it into JSON format
 app.use(express.json());
 
+app.post('/src',
+   // parseUserQuery,
```

Comment:
Excellent PR.

However, please fix the endpoint. This differs from the endpoint agreed upon in the post-standup meeting today. See the Excalidraw for additional notes.

Reply:
According to the "contract" document we made between the front end team (currently @CodeRepeater) and the back end team, this contract is `/api`.

That is, we want the URL to be `http://localhost:3000/api` not `http://localhost:3000/src`.

Now, we *did* move the `server.ts` file into it's own `src` directory during our Zoom. But, that is different than the API endpoint.